src/irc/ConnectionInstance.ts

```diff
@@ -188,7 +188,6 @@ export class ConnectionInstance {
                 return;
             }
             // do the callback
-            // eslint is usually confused about IArguments
```

Comment:
We've removed the comment but not the disable line?

Reply:
Nothing changed. The comment was outdated.
The JS variable `arguments` has the type `IArguments` but when collecting parameters as `...args` it's just an array.

Reply:
But below we have `fn.apply(fn, args as any);` which isn't doing ...args? Should we also fix that?

Reply:
```
function foo(...args) {
  //args is an Array<any>
}

function bar() {
  //arguments (defined in every function as part of the JS spec) is an IArguments
}
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments